<?xml version="1.0" encoding="UTF-8"?>
<<<<<<< HEAD
<testng-results ignored="0" total="3" passed="0" failed="3" skipped="0">
=======
<<<<<<< HEAD
<testng-results ignored="6" total="13" passed="4" failed="3" skipped="0">
=======
<testng-results ignored="5" total="12" passed="4" failed="3" skipped="0">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
  <reporter-output>
  </reporter-output>
<<<<<<< HEAD
  <suite started-at="2025-06-29T22:00:19 IST" name="Default suite" finished-at="2025-06-29T22:02:31 IST" duration-ms="131215">
=======
<<<<<<< HEAD
  <suite started-at="2025-06-29T20:31:05 IST" name="Default suite" finished-at="2025-06-29T20:31:36 IST" duration-ms="31126">
=======
  <suite started-at="2025-06-29T20:28:11 IST" name="Default suite" finished-at="2025-06-29T20:28:29 IST" duration-ms="18737">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
    <groups>
    </groups>
<<<<<<< HEAD
    <test started-at="2025-06-29T22:00:19 IST" name="Default test" finished-at="2025-06-29T22:02:31 IST" duration-ms="131215">
=======
<<<<<<< HEAD
    <test started-at="2025-06-29T20:31:05 IST" name="Default test" finished-at="2025-06-29T20:31:36 IST" duration-ms="31126">
=======
    <test started-at="2025-06-29T20:28:11 IST" name="Default test" finished-at="2025-06-29T20:28:29 IST" duration-ms="18737">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
      <class name="tests.TestScripts">
<<<<<<< HEAD
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:00:20 IST" name="setUp" finished-at="2025-06-29T22:00:24 IST" duration-ms="4768" status="PASS">
=======
<<<<<<< HEAD
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:05 IST" name="setUp" finished-at="2025-06-29T20:31:12 IST" duration-ms="6803" status="PASS">
=======
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:11 IST" name="setUp" finished-at="2025-06-29T20:28:16 IST" duration-ms="5577" status="PASS">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
<<<<<<< HEAD
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:00:24 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T22:00:24 IST" duration-ms="79" status="PASS">
=======
<<<<<<< HEAD
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:14 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:14 IST" duration-ms="71" status="PASS">
=======
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:17 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:17 IST" duration-ms="53" status="PASS">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
<<<<<<< HEAD
        <test-method signature="Doctor()[pri:6, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:00:24 IST" name="Doctor" finished-at="2025-06-29T22:00:40 IST" duration-ms="15749" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at pages.HomePage.consultDoctor(HomePage.java:214)
at tests.TestScripts.Doctor(TestScripts.java:74)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
=======
<<<<<<< HEAD
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:14 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:17 IST" duration-ms="3131" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@477021ee]]>
=======
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:17 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:19 IST" duration-ms="1525" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@740dcae3]]>
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
              </value>
            </param>
          </params>
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
          <reporter-output>
          </reporter-output>
<<<<<<< HEAD
        </test-method> <!-- Doctor -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:00:40 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T22:00:40 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="Range()[pri:7, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:00:40 IST" name="Range" finished-at="2025-06-29T22:01:54 IST" duration-ms="73502" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at pages.HomePage.notInRange(HomePage.java:243)
at tests.TestScripts.Range(TestScripts.java:80)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Range -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:01:54 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T22:01:54 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="Success()[pri:8, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:01:54 IST" name="Success" finished-at="2025-06-29T22:02:26 IST" duration-ms="32537" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.fail(Assert.java:116)
at pages.HomePage.success(HomePage.java:276)
at tests.TestScripts.Success(TestScripts.java:86)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Success -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.TestScripts@367ffa75]" started-at="2025-06-29T22:02:26 IST" name="tearDown" finished-at="2025-06-29T22:02:31 IST" duration-ms="4499" status="PASS">
=======
        </test-method> <!-- validateDiastolic -->
<<<<<<< HEAD
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:17 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:17 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:17 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:20 IST" duration-ms="2623" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@2180e789]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:20 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:20 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:20 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:22 IST" duration-ms="2395" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@5e9bbd9d]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:22 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:22 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:22 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:25 IST" duration-ms="2517" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@4b87074a]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:25 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:25 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:25 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:27 IST" duration-ms="2390" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@650c405c]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[100 DiastolicBP needs to fails but pass. expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 100 DiastolicBP needs to fails but pass. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.TestScripts.validateDiastolic(TestScripts.java:59)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:27 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:27 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:27 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:30 IST" duration-ms="2533" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@180b3819]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[101 DiastolicBP needs to fails but pass. expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 101 DiastolicBP needs to fails but pass. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.TestScripts.validateDiastolic(TestScripts.java:59)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:30 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:31:30 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:30 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:31:32 IST" duration-ms="2360" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@499ee966]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[150 DiastolicBP needs to fails but pass. expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 150 DiastolicBP needs to fails but pass. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.TestScripts.validateDiastolic(TestScripts.java:59)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.TestScripts@24269709]" started-at="2025-06-29T20:31:32 IST" name="tearDown" finished-at="2025-06-29T20:31:36 IST" duration-ms="4365" status="PASS">
=======
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:19 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:20 IST" duration-ms="1351" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@4e140497]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:19 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:19 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:20 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:20 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:20 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:22 IST" duration-ms="1404" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@481b2f10]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:22 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:22 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:22 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:23 IST" duration-ms="1137" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@2b6c7012]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:23 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:23 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:23 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:24 IST" duration-ms="1217" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@7ec95456]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.UnhandledAlertException">
            <message>
              <![CDATA[unexpected alert open: {Alert text : Provided BP values are not valid. Please Check Once Again}
  (Session info: chrome=137.0.7151.69): Provided BP values are not valid. Please Check Once Again
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [ec87eb07d118c560bd35db2ed50f8bb4, findElement {using=id, value=bp-diastolic}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2401015\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52601}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52601/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: ec87eb07d118c560bd35db2ed50f8bb4]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Provided BP values are not valid. Please Check Once Again}
  (Session info: chrome=137.0.7151.69): Provided BP values are not valid. Please Check Once Again
Build info: version: '4.32.0', revision: 'd17c8aa950'
System info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '21.0.5'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [ec87eb07d118c560bd35db2ed50f8bb4, findElement {using=id, value=bp-diastolic}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 137.0.7151.69, chrome: {chromedriverVersion: 137.0.7151.119 (e0ac9d12dff..., userDataDir: C:\Users\2401015\AppData\Lo...}, fedcm:accounts: true, goog:chromeOptions: {debuggerAddress: localhost:52601}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: windows, proxy: Proxy(), se:cdp: ws://localhost:52601/devtoo..., se:cdpVersion: 137.0.7151.69, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:extension:minPinLength: true, webauthn:extension:prf: true, webauthn:virtualAuthenticators: true}
Session ID: ec87eb07d118c560bd35db2ed50f8bb4
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:136)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:50)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:215)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:216)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:174)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:59)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:368)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:362)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:68)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at jdk.proxy2/jdk.proxy2.$Proxy16.getWrappedElement(Unknown Source)
at org.openqa.selenium.remote.WebElementToJsonConverter.apply(WebElementToJsonConverter.java:48)
at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:1024)
at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:682)
at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:474)
at pages.HomePage.verify(HomePage.java:53)
at pages.HomePage.diastolicBP(HomePage.java:124)
at tests.TestScripts.validateDiastolic(TestScripts.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.UnhandledAlertException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:24 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:24 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:24 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:25 IST" duration-ms="1096" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@9b23822]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[101 DiastolicBP needs to fails but pass. expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 101 DiastolicBP needs to fails but pass. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.TestScripts.validateDiastolic(TestScripts.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method signature="validateDiastolic([Ljava.lang.String;)[pri:4, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:25 IST" name="validateDiastolic" data-provider="filereader" finished-at="2025-06-29T20:28:26 IST" duration-ms="1286" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[[Ljava.lang.String;@740fefc6]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[150 DiastolicBP needs to fails but pass. expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 150 DiastolicBP needs to fails but pass. expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertFalse(Assert.java:79)
at tests.TestScripts.validateDiastolic(TestScripts.java:63)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateDiastolic -->
        <test-method is-config="true" signature="setUpPagesAndUtils()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:25 IST" name="setUpPagesAndUtils" finished-at="2025-06-29T20:28:25 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpPagesAndUtils -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:tests.TestScripts@5dda768f]" started-at="2025-06-29T20:28:26 IST" name="tearDown" finished-at="2025-06-29T20:28:29 IST" duration-ms="3103" status="PASS">
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
>>>>>>> branch 'main' of https://github.com/Gnanaprakash-S/NoSuchTeam.git
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- tests.TestScripts -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
